t/rosie-create/00-basic.t

```diff
@@ -87,14 +87,17 @@ title=this should never fail
 __INFO__
 run_pass "$TEST_KEY" rosie create <<<y
 file_cmp "$TEST_KEY.edit.out" "$TEST_KEY_BASE-edit.out" <<__INFO__
+description=
 owner=fred
 project=
+sub-project=
 title=
 
 # Make changes ABOVE these lines.
-# The "owner", "project" and "title" fields are compulsory.
+# The "owner", "project", and "title" fields are compulsory.
 # Any KEY=VALUE pairs can be added. Known fields include:
-# "access-list", "description", "sub-project", and "issue-list".
+# "access-list", "description", "period", "sub-project" 
+# and "type".
```

Comment:
The name overlaps with our metadata `type` (`type=integer`, etc). I think this is probably already handled by project, subproject, and title. We can add it later if not.